python_modules/libraries/dagster-dg/dagster_dg/utils.py

```diff
@@ -1,17 +1,23 @@
 import contextlib
+import hashlib
 import os
 import posixpath
 import re
 import subprocess
 import sys
+from fnmatch import fnmatch
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Final, Iterator, List, Mapping, Optional, Sequence, Union
 
 import click
 import jinja2
+from typing_extensions import TypeAlias
 
 from dagster_dg.version import __version__ as dagster_version
 
+# HASH is not publicly exported from hashlib, so alias here for only one type-ignore.
+Hash: TypeAlias = hashlib.HASH  # type: ignore
```

Comment:
should we be depending on it if that is true?

Reply:
I think it is _intended_ to be publicly exported but it just isn't by the rules of `pyright`, since it is imported from another module without a redundant alias. I think it's fine.